js/script.js

```diff
@@ -0,0 +1,109 @@
+const library = [];
+
+function Book(author, title, pages, read = false) {
+  this.author = author;
+  this.title = title;
+  this.pages = pages;
+  this.read = read;
+}
+
+function addBookToLibrary(book) {
+  library.push(book);
+}
+
+function removeBookFromLibrary(bookIndex) {
+  library.splice(bookIndex, 1);
+}
+
+function changeReadStatus(bookIndex) {
+  if (library[bookIndex].read) {
+    library[bookIndex].read = false;
+  } else {
+    library[bookIndex].read = true;
+  }
+}
+
+const addNewForm = document.getElementById('new-book-form');
+const bookList = document.querySelector('.book-list');
+const formToggleBtn = document.getElementById('form-toggle-btn');
+const formContainer = document.getElementById('new-book-form-container');
+
+formToggleBtn.addEventListener('click', () => {
+  formContainer.classList.toggle('d-none');
+});
+
+
+function render(bookArray) {
+  bookList.innerHTML = '';
+  bookArray.forEach((book, index) => {
+    // eslint-disable-next-line no-use-before-define
+    bookList.appendChild(createCard(book, index));
+  });
+}
+
+function createCard(book, id) {
```

Comment:
Let's separate the DOM from application logic, you can create another file for DOM.

![image](https://user-images.githubusercontent.com/31897434/73420341-70e18b80-4322-11ea-840a-fa14e04b592b.png)

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59623451/